ocaml/message-switch/core_test/concur-rpc-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+
+SPATH=${TMPDIR:-/tmp}/sock_p
```

Comment:
You might want to use `$$` (the shell's PID) to make the file names unique and avoid collision with another instance. This code also only works in a specific directory - so might do something like

```
test -x ./server_unix_main.exe || exit 1
```
```
trap "cleanup" TERM INT

function cleanup {
 rm -rf ${SWITCHPATH}
}
```